lib/Data/Validate/Sanctions.pm

```diff
@@ -54,25 +66,52 @@ sub get_sanctioned_info {    ## no critic (RequireArgUnpacking)
     my $data = $self->_load_data();
 
     # prepare list of possible variants of names: LastnameFirstname and FirstnameLastname
+    my @full_name = $last_name ? ($first_name, $last_name) : $first_name;
```

Comment:
```suggestion
    my @full_name = ($first_name, $last_name || ());
```

Reply:
This gives an error :/ all my tests fail